### PR TITLE
[query] Update prometheus.md

### PR DESCRIPTION
**Pull request by bianap**
tagOptions and idScheme keys are missing from the example configuration based on m3coordinator-cluster-template.yml.

**What this PR does / why we need it**:

Updating the prometheus.md, adding the missing part of the configuration. 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note
NONE
```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note
NONE
```

### CODE REVIEW

File: docs/integrations/prometheus.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ metrics:
   sanitization: prometheus
   samplingRate: 1.0
   extended: none
+  
+  tagOptions:
```
Comment (benraskin92) [inline docs/integrations/prometheus.md]: I think `tagOptions` is indented one too far.

File: docs/integrations/prometheus.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ metrics:
...
+  idScheme: quoted
```
Comment (benraskin92) [inline docs/integrations/prometheus.md]: This should be under `tagOptions`.

File: docs/integrations/prometheus.md (hunk 1 above)
Comment (bianap) [reply docs/integrations/prometheus.md]: Thank you, I corrected it in a new commit.

File: docs/integrations/prometheus.md (hunk 2 above)
Comment (bianap) [reply docs/integrations/prometheus.md]: Thank you, I corrected it in a new commit.

**Pull request merged**